.gitignore

```diff
@@ -13,4 +13,4 @@ dist/
 AUTHORS
 ChangeLog
 alembic.ini
-
+/.vscode
```

Comment:
Just a nitpick - this isn't relevant to the PR.

Reply:
Not really, and I can move to a separate PR if you want ;-)